webhookbot/webhookbot/http.go

```diff
@@ -55,7 +72,7 @@ func (h *HTTPSrv) handleHook(w http.ResponseWriter, r *http.Request) {
 	}
 	msg, err := h.getMessage(r)
 	if err != nil {
-		h.Debug("handleHook: failed to find message: %s", err)
+		h.Errorf("handleHook: failed to find message: %s", err)
```

Comment:
I"m not sure about this, seems like we will pick up a lot of noise

Reply:
I wanted to see what came through here for a bit if there was anything useful and I'll turn it off if not